src/components/authButton/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FunctionComponent } from "react";
```

Comment:
We might want to move this into a 'variant' as Tony showed today - might be cleaner. I'm also okay with merging this in rn however and making a bug ticket for this in the future - your call @Aphexis